src/masoniteorm/commands/MigrateRefreshCommand.py

```diff
@@ -9,12 +9,22 @@ class MigrateRefreshCommand(Command):
     migrate:refresh
         {--c|connection=default : The connection you want to run migrations on}
         {--d|directory=databases/migrations : The location of the migration directory}
+        {--s|seed=? : Wheter seeding should be run after refresh migrations and which seedeer should be ran}
```

Comment:
I would update the description to:
```suggestion
        {--s|seed=? : Seed database after refresh. The seeder to be ran can be provided in argument}
```